src/vsr/replica.zig

```diff
@@ -6647,6 +6985,15 @@ pub fn ReplicaType(
                 read.checksum,
                 read.block_type,
             });
+
+            if (self.solo()) @panic("grid is corrupt");
+
+            if (!self.grid_repair_message_timeout.ticking) {
+                assert(self.grid.read_faulty_queue.count == 1);
+
+                self.grid_repair_message_timeout.start();
+                self.on_grid_repair_message_timeout();
```

Comment:
Hm, invoking the timeout manually is not something we do elsewhere, it is a bit surprising. 

Perhaps we should wait until the next replica tick? Right now we'll immediately send a request for a single block, but it doesn't seem unreasonable that more than one block would fail at the same time?